_docs-unreleased/troubleshooting/basic.md

```diff
@@ -139,6 +139,13 @@ Note that the use of `PrintInfo` is an administrative tool and can only
 by used by someone who can access the underlying Accumulo data. It
 does not provide the normal access controls in Accumulo.
 
+## Ingest
+
+**Why does my ingest rate periodically go down during heavy ingest?**
+
+Periods of zero or low ingest rates are typically caused by Java garbage collection pauses in tablet servers. This problem
```

Comment:
I wouldn't say "typically" here. It could be caused by many things.

s/are typically/can be/